torchdynamo/variables/tensor.py

```diff
@@ -113,6 +113,12 @@ def create(cls, tx, proxy, example_value=None, nnmodule=None, **options):
             def wrap_fake_exception(func):
                 return func()
 
+        if (
```

Comment:
let me know if there's a list or something somewhere else I should be putting this (or starting)

Reply:
YOu could just add it here - https://github.com/pytorch/torchdynamo/blob/main/torchdynamo/allowed_functions.py#L66

Reply:
This won't make an unimplemented though - this just elides it out - I actually dislike this list because it just removes func silently. Perhaps worth discussing at some point.

Reply:
Potentially true but leaving for a separate PR

Reply:
Does this graph break? If it does not graph break, does this correctly go to the interpreter? Or do we just remove it silently? If it does not go to the interpreter, this will cause correctness issues.

Reply:
uh, i believe so, on stats you get 
```
graph_break [('inline in skipfiles: get_rng_state /scratch/eellison/work/pytorch/torch/random.py', 1), ('inline in skipfiles: set_rng_state /scratch/eellison/work/pytorch/torch/random.py', 1)]
```
and 
the frame_count is 2

Reply:
Perfect, thank you :)

Reply:
> This won't make an unimplemented though - this just elides it out - I actually dislike this list because it just removes func silently. Perhaps worth discussing at some point.

@voznesenskym Oh I did not know that. Yes, lets clean it up if this needs refactoring. I will also look deeply to see if I am missing anything.

Reply:
I think its fine since it becomes an inline in skipfiles above, as per @eellison comment, which makes this fine!